rust/tw_evm/src/transaction/user_operation_v0_7.rs

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright © 2017 Trust Wallet.
+
+use crate::abi::encode::encode_tokens;
+use crate::abi::non_empty_array::NonEmptyBytes;
+use crate::abi::token::Token;
+use crate::address::Address;
+use crate::transaction::signature::Signature;
+use crate::transaction::{SignedTransaction, TransactionCommon, UnsignedTransaction};
+use serde::Serialize;
+use tw_coin_entry::error::prelude::*;
+use tw_encoding::hex;
+use tw_hash::sha3::keccak256;
+use tw_hash::H256;
+use tw_memory::Data;
+use tw_number::U256;
+
+pub struct UserOperationV0_7 {
```

Comment:
Can we strictly align the field names with the EIP-4337 standard?
https://eips.ethereum.org/EIPS/eip-4337#useroperation